heron/tmaster/src/cpp/manager/tmaster.cpp

```diff
@@ -364,7 +364,10 @@ proto::system::Status* TMaster::RegisterStMgr(
     // First check to see if that other guy has timed out
     if (!stmgrs_[stmgr_id]->TimedOut()) {
       // we reject the new guy
-      LOG(ERROR) << "Another stmgr exists with the same id and it hasn't timed out" << std::endl;
+      LOG(ERROR) << "Another stmgr exists at "
```

Comment:
@congwang It is better to have a log having all necessary info self-contained, than relying on earlier logs. It is possible that people mistakenly hurt this log without knowing it.